app/assets/stylesheets/common/base/user.scss

```diff
@@ -789,3 +789,24 @@
     grid-column: 1 / span 2;
   }
 }
+
+.notification-schedule {
+  .instruction {
+    margin-top: 12px;
+    margin-bottom: 10px;
+  }
+
+  .notification-schedule-table {
+    tbody {
```

Comment:
please do direct styling on html element, this is how we end up in !important hell later